src/notifications/lipsticknotification.h

```diff
@@ -46,6 +46,7 @@ class LIPSTICK_EXPORT LipstickNotification : public QObject
     Q_PROPERTY(int priority READ priority NOTIFY priorityChanged)
     Q_PROPERTY(QString category READ category NOTIFY categoryChanged)
     Q_PROPERTY(bool userRemovable READ isUserRemovable NOTIFY userRemovableChanged)
+    Q_PROPERTY(QVariantList remoteActions READ remoteActions)
```

Comment:
Yes, practically speaking there's no possibility of the actions changing - to date there's only been one (remote) action supported.  I could support changing them after creation, but it complicates the manger, the originator and the renderer of the notification, so I would rather have a known use case before doing that.